docs/weighting/poiAndTransitNodeWeighting_en.md

```diff
@@ -0,0 +1,346 @@
+- $\beta_2$ governs the exponential decay, which becomes more influential for longer times/distances, representing regional sensitivity where accessibility drops off more gradually beyond a certain threshold.
+
+This dual nature allows the function to model both immediate, sharp decay for nearby opportunities and smoother decay for distant ones, as validated in pedestrian accessibility studies (Vale & Pereira, 2017). Values are mode-specific; for walking, typical ranges are $\beta_1 = 0.5-1.5$ and $\beta_2 = 0.1-0.3$.
+
+### Gamma function (NCHRP 716 recommended standard)
+
+The **gamma function** is recommended by NCHRP Reports 365 and 716 as the standard for travel time-based gravity models in four-step transportation planning:
+
+**Formula:**
+
+$$f(t_{ij}) = a \times t_{ij}^{-b} \times \exp(-c \times t_{ij})$$
+
+**Where:**
+- $t_{ij}$ = travel time (minutes) from zone $i$ to zone $j$
+- $a$ = scaling parameter ($> 0$)
+- $b$ = power parameter ($> 0$)
+- $c$ = exponential decay parameter ($> 0$)
+
+**Typical coefficient ranges by trip purpose (Medium MPO):**
+
+| Trip Purpose | a | b | c | Application |
+|--------------|---|---|---|-------------|
+| Home-Based Work | 2000-6000 | 0.8-1.2 | 0.05-0.10 | Work commute trips |
+| Home-Based Non-Work | 1500-3500 | 0.5-0.9 | 0.08-0.15 | Shopping, social, personal |
+| Non-Home-Based | 1000-2500 | 0.4-0.8 | 0.10-0.18 | Chained trips |
+
+**Example calculation:** For home-based work trips with $t=15$ minutes using parameters $a=5280$, $b=0.926$, $c=0.087$:
+
+$$\begin{align*}
+f(15) &= 5280 \times 15^{-0.926} \times \exp(-0.087 \times 15) \\
+      &= 5280 \times 0.0748 \times 0.267 \\
+      &= 105.4 \text{ (friction factor)}
+\end{align*}$$
+
+## Distance and time decay parameters across modes and purposes
+
+Empirical studies show $\beta$ values vary dramatically by mode and trip purpose, quantifying that pedestrians are extremely distance-sensitive while drivers tolerate longer trips.
+
+### Distance-based parameters
+
+| Mode/Purpose | Distance Decay ($\beta$) | Function Type | Data Source |
+|--------------|-------------------|---------------|-------------|
+| **Walking** |
+| - Work | 1.683 | Power | Iacono et al. (2008) |
+| - Shopping | 2.106 | Power | Iacono et al. (2008) |
+| - Entertainment | 0.769 | Power | Iacono et al. (2008) |
+| - To Transit (network) | 0.00217 | Exponential | Bartzokas-Tsiompras (2019) |
+| **Transit** |
+| - Local Bus | 0.186 | Power | Iacono et al. (2008) |
+| - Light Rail | 0.046 | Power | Iacono et al. (2008) |
+| - Express Bus | 0.060 | Power | Iacono et al. (2008) |
+| **Auto** |
+| - Work | 0.09 | Power | Iacono et al. (2008) |
+| - Shopping | 0.118 | Power | Iacono et al. (2008) |
+
+### Travel time-based parameters
+
+| Mode | $\beta$ (time-based) | Function Type | Interpretation |
+|------|----------------|---------------|----------------|
+| Walking | 0.20-0.30 | Exponential | Very steep decay (high time sensitivity) |
```

Comment:
You had reference for the beta in the previous section, but not here, why?

Reply:
added